app/views/ListView.scala.html

```diff
@@ -15,79 +15,107 @@
  *@
 
 @this(
-    layout: templates.Layout,
-    formHelper: FormWithCSRF,
-    govukErrorSummary: GovukErrorSummary,
-    govukButton: GovukButton,
-    govukRadios: GovukRadios,
-    hmrcListWithActions: HmrcListWithActions,
-    govukInsetText: GovukInsetText,
-    govukPagination: GovukPagination
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukButton: GovukButton,
+        govukRadios: GovukRadios,
+        hmrcListWithActions: HmrcListWithActions,
+        govukInsetText: GovukInsetText,
+        govukPagination: GovukPagination
 )
 @(form: Form[Boolean], viewmodel: FormPageViewModel[ListViewModel])(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, viewmodel.title.toMessage)) {
+    @layout(pageTitle = title(form, viewmodel.title.toMessage)) {
```

Comment:
Do we need a selection here for the view-only title? (as it is currently duplicated and if someone in future passes in separate values from the controller)

Reply:
added